portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,130 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter; //to out html
+//Webservlet stuff
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+//for datastore 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+//for querying datastore
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+//for user API
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+//formating json
+import com.google.gson.Gson;
+
+@WebServlet("/loginstat")
+public class LoginServlet extends HttpServlet {
+
+     public final class Login {
```

Comment:
Hmm not sure if `Login` conveys what this class actually represents? This seems to be more like a `LoginStatus`? If that is the case, using more apt naming even if longer such as `LoginStatus` will make this a bit easier for readers to understand